api/organization.go

```diff
@@ -6,10 +6,9 @@ import (
 )
 
 type Organization struct {
-	ID      uid.ID `json:"id"`
-	Name    string `json:"name"`
-	Created Time   `json:"created"`
-	Updated Time   `json:"updated"`
```

Comment:
What's the reason for removing these? Since orgs in the API are really only visible to support-admins, it seems like it would be pretty valuable to be able to see when an org was created or update.